README.md

```diff
@@ -72,6 +72,36 @@ $ clojure -M:run
 
 For a more detailed setup of a dev environment for Metabase, check out our [Developers Guide](./docs/developers-guide/start.md).
 
+### MAGE - Development Automation
```

Comment:
Do you think this is worth putting in the project-level README? It's mostly specific to BE development. I could imagine a new page in the [developers guide](https://github.com/metabase/metabase/blob/master/docs/developers-guide/start.md) or something

Reply:
I think eventually this will be the main entrypoint for everything so we have a BE<->FE shared idea of what's going on. I can wait until that point to add it into the readme if you think that's a better idea.

Reply:
Ok. I'm not personally opposed to having this in the project-level README but might be worth getting a broader audience in case anyone else has an opinion 🤷

Reply:
If we want to make this a main entry point to every task I think it's good to have it at the top level?